spec/models/custom_button_spec.rb

```diff
@@ -317,6 +317,7 @@
             :event_type  => 'button.trigger.start',
             :user_id     => user.id
           )
+          expect(CustomButtonEvent.first.timestamp).not_to be_nil
```

Comment:
Would be good to use `Timecop.freeze` so we can validate this time instead of only checking that it is not `nil`.